components/host-dashboard/HostDashboardReports.js

```diff
@@ -171,7 +171,7 @@ const HostDashboardReports = ({ hostSlug }) => {
   const [dateInterval, setDateInterval] = React.useState(getDefaultDateInterval);
   const [collectives, setCollectives] = React.useState(null);
   const dateFrom = simpleDateToISOString(dateInterval?.from, false, dateInterval?.timezoneType);
-  const dateTo = simpleDateToISOString(dateInterval?.to, true, dateInterval?.timezoneType);
+  const dateTo = simpleDateToISOString(dateInterval?.to, false, dateInterval?.timezoneType);
```

Comment:
Why that change?

Reply:
Here I thought that the end date should be start of the date not end of the day, but from you feedback on the other PR I realize that this isn't what is intended and we should include the whole end date so I've changed it back.